src/containers/singleEvent/index.tsx

```diff
@@ -52,7 +112,21 @@ const SingleEvent: React.FC<SingleEventProps> = ({ events }) => {
       const event = eventsList.result.filter((e) => e.id === id);
 
       if (event.length > 0) {
-        return <EventDetails {...event[0]} />;
+        return (
+          <>
+            {privilegeLevel === PrivilegeLevel.ADMIN ? (
+              <AdminActionButtonList>
+                <GreenButton to={'/events/' + id}>Edit</GreenButton>
```

Comment:
You could make the `/events/` part a constant since it's used multiple times. Good to stay away from magic strings, especially if they're used multiple times